front/screens/Onboarding.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+import { Button, Image, ImageSourcePropType, StyleSheet, Text, Dimensions } from 'react-native';
+import { View } from '../components/Themed';
+import { SwiperFlatList } from 'react-native-swiper-flatlist';
+import Colors from '../constants/Colors';
+
+interface Props {
+  navigation: any
+}
+interface SlideView {
+  title: string,
+  image: ImageSourcePropType,
+  description: string,
+}
+
+export class Onboarding extends React.Component<Props> {
```

Comment:
```suggestion
export const Onboarding: FC<Props> {
```

Reply:
Passer ce composant en fonctionnel : 
- ajouter `const` devant `appName` et `slideViews` et `getSlideViews`
- retirer les `this.` 
- utiliser `return` plutôt que `render` pour retourner le JSX